src/brsTypes/components/RoAppInfo.ts

```diff
@@ -0,0 +1,148 @@
+import { BrsBoolean, BrsString, BrsValue, ValueKind } from "../BrsType";
+import { BrsComponent } from "./BrsComponent";
+import { BrsType } from "..";
+import { Callable, StdlibArgument } from "../Callable";
+import { Interpreter } from "../../interpreter";
+import * as PP from "../../preprocessor";
+
+export class RoAppInfo extends BrsComponent implements BrsValue {
+    readonly kind = ValueKind.Object;
+
+    private static manifest = PP.getManifestSync(process.cwd());
```

Comment:
Passing `process.cwd()` will introduce some very subtle bugs when the root directory for `brs` isn't `process.cwd()` — it often isn't in the end-to-end tests in this project!

Since we parse the manifest as basically [the first step of this project](https://github.com/sjbarag/brs/blob/35d45dfe276b06e137e681094076a3986bab175d/src/index.ts#L56-L59) (before brightscript files are even loaded!) I'm curious if we could re-use the parsed manifest, possibly by storing it as a property on the `Interpreter` instance that [gets created early-on](https://github.com/sjbarag/brs/blob/35d45dfe276b06e137e681094076a3986bab175d/src/index.ts#L144-L148)?

Reply:
Thanks @sjbarag 
I've introduced the `manifest` as property on the `Interpreter` instance as you suggested. So, we no longer re-parse manifest and don't use `process.cwd()`.